docs: Add DB name to backup-and-restore (#13244)
immich-app/immich: docs/docs/administration/backup-and-restore.md

Then you can restore with the same command but pointed at the latest dump.

```bash title='Automated Restore'

> # Be sure to check the username and database name if you changed them from defaults

Reviewer: ```suggestion
# Be sure to check the username if you changed it from default
```
